api.md

```diff
@@ -621,20 +621,18 @@ when creating distributed hypertables.
 
 |Name|Description|
 |---|---|
-| `partitioning_column` | Name of an additional column to partition by. If provided, the `number_partitions` argument must also be provided. |
-| `number_partitions` | Number of hash partitions to use for `partitioning_column`. Must be > 0. |
+| `partitioning_column` | Name of an additional column to partition by. |
+| `number_partitions` | Number of hash partitions to use for `partitioning_column`. Must be > 0. Default is the number of `data_nodes`. |
+| `associated_schema_name` | Name of the schema for internal hypertable tables. Default is "_timescaledb_internal". |
+| `associated_table_prefix` | Prefix for internal hypertable chunk names. Default is "_hyper". |
 | `chunk_time_interval` | Interval in event time that each chunk covers. Must be > 0. As of TimescaleDB v0.11.0, default is 7 days, unless adaptive chunking (DEPRECATED)  is enabled, in which case the interval starts at 1 day. For previous versions, default is 1 month. |
 | `create_default_indexes` | Boolean whether to create default indexes on time/partitioning columns. Default is TRUE. |
 | `if_not_exists` | Boolean whether to print warning if table already converted to hypertable or raise exception. Default is FALSE. |
 | `partitioning_func` | The function to use for calculating a value's partition.|
-| `associated_schema_name` | Name of the schema for internal hypertable tables. Default is "_timescaledb_internal". |
-| `associated_table_prefix` | Prefix for internal hypertable chunk names. Default is "_hyper". |
-| `migrate_data` | Set to `true` to migrate any existing `main_table` data to chunks in the new hypertable. A non-empty table will generate an error without this option. Note that, for large tables, the migration might take a long time. Defaults to false. |
+| `migrate_data` | Set to `true` to migrate any existing `relation` data to chunks in the new hypertable. A non-empty table will generate an error without this option. Note that, for large tables, the migration might take a long time. Defaults to FALSE. |
```

Comment:
Not sure if this is intentional. You are using TRUE above, but `true` here.
```suggestion
| `migrate_data` | Set to TRUE to migrate any existing `relation` data to chunks in the new hypertable. A non-empty table will generate an error without this option. Note that, for large tables, the migration might take a long time. Defaults to FALSE. |
```

Reply:
@mkindahl It is not me, it was there. I homogenised only places where it wasn't code.

Reply:
We need decision about consistent way of using Boolean constants, since it is all possible cases in the file.

Reply:
Pending internal poll, I'll post the decision =)

Reply:
Homogenise a little bit, based on the current discussion. However, I suggest to do such text refactoring separately.